### ISSUE TITLE
Support for bits and chisels blocks?

### ISSUE THREAD

**Issue by netcore10** [BlueMap-Minecraft/BlueMap#359]
**Is your feature request related to a problem? Please describe.**
Bit's and chisel blocks don't show up on the map, is there a chance some way of implementing it? or is it's way of rendering just not very good? If anything could you atleast get it's base texture "stone" "birch" etc and use that?

![image](https://user-images.githubusercontent.com/181536/199739910-f29d23be-ec08-4079-9409-5e5d4b2c39c8.png)

**Comment by TBlueF** [comment]
BlueMap's rendering is based on Resources like in a Resource-Pack ... Mods that add "normal" new blocks have those resources included, which BlueMap is using to render them. Bit's and chisel is doing A LOT of magic with their blocks, and BlueMap will not be able to support that.

You can in theory make a resourcepack that adds custom models to blocks based on their block-state. So, depending on how much info Bits and Chisels stores in their BlockState, you can make a resourcepack that represents them as a textured cube or something like that.. But if Bits and Chisels doesn't add their "base-texture" to the BlockState, but rather stores that in the TileEntity-Data of their blocks, then BlueMap won't be able to read this kind of information.

**Tl;dr:** Right now, there is no feasible way to add support for Bits and Chisels and there is and will also nothing like this be explicitly planned, sorry :)

**Comment by netcore10** [comment]
That's fine, i really appreciate you even taking the time to look!
